File: programacion/tema08/ejemplo01/src/main/java/com/example/ejemplo01/PrincipalController.java
package com.example.ejemplo01;

import javafx.fxml.FXML;
import javafx.scene.control.Label;

public class PrincipalController {
    @FXML
    private lateinit var txtNombre: textField;

    @FXML
    protected void onHelloButtonClick() {
        welcomeText.setText("Welcome to JavaFX Application!");
    }

}

File: programacion/tema08/ejemplo2/src/main/kotlin/com/example/ejemplo2/PrincipalController.kt
package com.example.ejemplo2

import javafx.fxml.FXML
import javafx.scene.control.Label

class PrincipalController {
    @FXML
    private lateinit var welcomeText: Label

    @FXML
    private fun onHelloButtonClick() {
        welcomeText.text = "Welcome to JavaFX Application!"
    }
}